.eslintrc.js

```diff
@@ -2,6 +2,7 @@ const clientESLintConfig = require('./config/eslint');
 
 module.exports = Object.assign({}, clientESLintConfig, {
   env: Object.assign({}, clientESLintConfig.env, {
-    node: true
+    node: true,
+    jest: true,
```

Comment:
Let’s just keep it enabled by default. I don’t think it’s a big deal. It’s not perfect but I don’t think most people would be somehow bitten by `describe` “getting through” the linter in regular files.